src/SokuMod/Connection.cpp

```diff
@@ -13,6 +13,7 @@ std::mutex logMutex;
 #include <Exceptions.hpp>
 #include "Connection.hpp"
 #include "getPublicIp.hpp"
+#include "ipv6map_extern.hpp"
```

Comment:
Where is that file? It's not included in the PR

Reply:
ummm i forgot to add them.